src/components/components.module.ts

```diff
@@ -4,19 +4,25 @@ import { IonicModule } from 'ionic-angular';
 import { JournalHeaderComponent } from './journal-header/journal-header';
 import { TestSummaryComponent } from './test-summary/test-summary';
 import { ButtonCardComponent } from './button-card/button-card';
+import { HeaderComponent } from './header/header';
 import { TestSummarySectionComponent } from './test-summary/test-summary-section/test-summary-section';
+
 @NgModule({
   declarations: [
     PageHeaderComponent,
     JournalHeaderComponent,
+    HeaderComponent,
     TestSummaryComponent,
     ButtonCardComponent,
     TestSummarySectionComponent
   ],
-  imports: [IonicModule.forRoot(PageHeaderComponent)],
+  imports: [
+    IonicModule.forRoot(PageHeaderComponent)
+  ],
```

Comment:
Did the prettier plugin change this? Using the current config, it should be all on one line.

Reply:
No - I did, making the arrays consistent...I can change it back!

Reply:
no worries, I think it's fine. I was just wondering if prettier is actually doing what it should be for everyone. We can change the config if this is the format we want.